app/docs/md/learn/components/image.md

```diff
@@ -0,0 +1,176 @@
+
+#### Attributes
+
+<dl class='pl2 pl4-lg'>
+  <dt class='font-bold'><code>format</code></dt>
+  <dd>The desired format for the generated image. Available formats are <code>jpg</code>, <code>png</code>, <code>gif</code>, <code>webp</code>, and <code>avif</code>. Defaults to the source image format.
+  <doc-callout level='caution' class='mt0'>
+
+  Not all browsers support all image formats — see [MDN’s image filetype and format guide](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types) for reference.
+
+  </doc-callout></dd>
+
+  <dt class='font-bold mt0'><code>media</code></dt>
+  <dd>The <a href='https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries#syntax'>media condition</a> under which the generated image should be displayed. If a media condition is not provided, the image will be rendered under all media conditions, unless a variant with a matching media condition precedes it.
+  <doc-callout level='caution' class='mt0'>
+
+  Due to [the design of the underlying specification](https://html.spec.whatwg.org/multipage/images.html#selecting-an-image-source), the order of media conditions is important: unlike in CSS, the browser will use the _first_  matching media condition it encounters, rather than examining each media condition for the best fit.
+
+  </doc-callout>
+
+  <doc-callout level='tip' class='mt0'>
+
+  As a safety net, where media conditions are all of type `min-width` or `max-width`, these will be reordered into the expected sequence by the component (descending for `min-width`, ascending for `max-width`). The authored order will persist in the generated output if other media features (or multiple media features) are used.
+
+  </doc-callout></dd>
+
+  <dt class='font-bold mt0'><code>quality</code></dt>
+  <dd>The quality setting controls the amount of compression to apply to the generated image. Acceptsa numeric string from <code>0</code> to <code>100</code>. Higher numbers result in higher quality and less compression, at the expense of larger file size. Defaults to full quality. TODO: which formats is this available for?</dd>
+
+  <dt class='font-bold mt0'><code>width</code></dt>
+  <dd>The desired <a href='https://developer.mozilla.org/en-US/docs/Glossary/Intrinsic_Size'>intrinsic width</a> of the generated image. Accepts a unitless numeric string, to specify the width in pixels. Defaults to the source image width.
+
+  <doc-callout level='info' class='mt0'>
+
+  Generated images will never be upscaled. If a provided `width` exceeds your source image’s width, it will be replaced by the source width.
+
+  </doc-callout>
+  </dd>
+
+</dl>
+
+## Examples
+
+### Basic optimization
+
+<doc-code>
+
+```html
+<enhance-image src='/_public/axol.png' alt='Illustration of Axol, the friendly axolotl that is Enhance’s mascot.'>
```

Comment:
Pedantic but please put the attributes on separate lines if you need to horizontally scroll the code block.